app/frontend/src/components/EditUserLocationMap.tsx

```diff
@@ -77,9 +77,15 @@ export default function EditUserLocationMap({
 
     map.current!.getCanvas().style.cursor = "grab";
 
-    const moveEvent = (e: MapMouseEvent | MapTouchEvent) => onCircleMove(e);
-    map.current!.on("mousemove", moveEvent);
-    map.current!.once("mouseup", (e) => onCircleUp(e, moveEvent));
+    if (e.type === "touchstart") {
+      const touchMoveEvent = (e: MapTouchEvent) => onCircleMove(e);
```

Comment:
I think it might be clearer to name these `handleTouchMove` and `handleMouseMove` instead of `touchMoveEvent` and `moveEvent`? It's a bit confusing to read an event being attached on an event 🤔 

And maybe do the same for the `onCircleUp` and `onHandleUp` and name the second parameter `handleMove` or something similar.

Reply:
I can do `handleTouchMove`/`handleMouseMove`  but I wanted to avoid the others because the you get `handleHandleDown`....

Reply:
Oh sorry I wasn't being clearer, I meant the variable name in the parameter of `onCircleUp`/`onHandleUp`. You can still rename the second parameter so it reads something like:
```typescript
const onCircleUp = (
    e: MapMouseEvent | MapTouchEvent,
    handleMove: (x: any) => void
  )
```
?